src/xcode/ENA/ENA/Source/Scenes/AppInformation/AppInformationViewController+DynamicTableViewModel.swift

```diff
@@ -78,7 +78,7 @@ private extension DynamicCell {
 private extension DynamicAction {
 	static var safari: Self {
 		.execute { viewController in
-			WebPageHelper.showWebPage(from: viewController)
+			WebPageHelper.showWebPage(from: viewController, urlString: AppStrings.SafariView.targetURL)
```

Comment:
Why is this URL here necessary? at least from what I can see it points to this

```swift 
	enum SafariView {
		static let targetURL = NSLocalizedString("safari_corona_website", comment: "")
	}
```

Reply:
the url is different depending on the language

Reply:
ah, I see now. But why wouldn't we move this to the AppStrings file?